pytools/test/test_persistent_dict.py

```diff
@@ -202,37 +202,6 @@ def test_persistent_dict_synchronization():
         shutil.rmtree(tmpdir)
 
 
-def test_persistent_dict_cache_collisions():
```

Comment:
For each deleted test, could you explain why they're no longer relevant?

Reply:
367683c & 7028e6b restore the collision tests.